pytorch_lightning/callbacks/progress.py

```diff
@@ -43,8 +42,36 @@ class tqdm(_tqdm):
     Custom tqdm progressbar where we append 0 to floating points/strings to prevent the progress bar from flickering
     """
 
-    @staticmethod
```

Comment:
You removed the `staticmethod` annotation

Reply:
added back !